docs/config.rst

```diff
@@ -18,11 +18,11 @@ Sample Configuration
             document_managers:
                 default:
                     mappings:
-                        AcmeDemoBundle: ~
-                    filters:
-                        filter-name:
-                            class: Class\Example\Filter\ODM\ExampleFilter
-                            enabled: true
+                        App:
+                            is_bundle: false
```

Comment:
The `is_bundle` and `alias` configs are not required for a default setup. I update the recipe: https://github.com/symfony/recipes-contrib/pull/1582